index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
I'm afraid I can't add any constructive feedback on the code as I am too much of a noob.
I also can't check the deployed site for visual appearance. And for that reason, I think the only suggestion would be to get a basic ugly version up and running first, then at least you have something to deploy (for the sake of this hackathon).

Reply:
ah, just realised you may have added a link/screenshot elsewhere =)